packages/xchain-bitcoincash/src/utils.ts

```diff
@@ -141,13 +141,24 @@ export const bchNetwork = (network: Network): BCHNetwork => {
 
 /**
  * Get address prefix based on the network.
- * Convert to Legacy Address.
  *
  * @param {string} network
  * @returns {string} The address prefix based on the network.
  *
  **/
-export const getPrefix = (network: string) => (network === 'testnet' ? 'm' : '1')
+export const getPrefix = (network: string) => (network === 'testnet' ? 'bchtest:' : 'bitcoincash:')
+
+/**
+ * Strips bchtest or bitcoincash prefix from address
+ *
+ * @param {Address} address
+ * @returns {Address} The address with prefix removed
+ *
+ */
+export const stripPrefix = (address: Address, network: Network): Address => {
+  const prefix = getPrefix(network)
+  return address.indexOf(prefix) === 0 ? address.substr(prefix.length, address.length) : address
+}
```

Comment:
@stuartwk _Just if you want_, here is shorthand using RegExt (no need to pass `Network` btw)
```ts
export const stripPrefix = (address: Address, network: Network): Address => address.replace(/(bchtest:|bitcoincash:)/, "")
```


```
"bchtest:qpd7jmj0hltgxux06v9d9u6933vq7zd0kyjlapya0g".replace(/(bchtest:|bitcoincash:)/, "")
// output: qpd7jmj0hltgxux06v9d9u6933vq7zd0kyjlapya0g

"bitcoincash:qp4kjpk684c3d9qjk5a37vl2xn86wxl0f5j2ru0dajg".replace(/(bchtest:|bitcoincash:)/, "") 
// output: qp4kjpk684c3d9qjk5a37vl2xn86wxl0f5j2ru0dajg
```

Reply:
Nice and clean, will updated 👍